src/tests/Interop/COM/NETClients/IDispatch/Program.cs

```diff
@@ -215,6 +215,65 @@ System.Collections.Generic.IEnumerable<int> GetEnumerable(System.Collections.IEn
             }
         }
 
+        static void Validate_ValueCoerce_ReturnToManaged()
+        {
+            var dispatchCoerceTesting = (DispatchCoerceTesting)new DispatchCoerceTestingClass();
+
+            Console.WriteLine($"Calling {nameof(DispatchCoerceTesting.ReturnToManaged)} ...");
+            
+            // Supported types
+            VarEnum[] supportedTypes =
+            {
+                VarEnum.VT_I2,
+                VarEnum.VT_I4,
+                VarEnum.VT_R4,
+                VarEnum.VT_R8,
+                VarEnum.VT_CY,
+                VarEnum.VT_DATE,
+                VarEnum.VT_BSTR,
+                VarEnum.VT_ERROR,
+                VarEnum.VT_BOOL,
+                VarEnum.VT_DECIMAL,
+            };
+
+            foreach (VarEnum vt in supportedTypes)
+            {
+                Console.WriteLine($"Converting {vt} to int should be supported.");
+                int result = dispatchCoerceTesting.ReturnToManaged((short)vt);
+                Assert.NotEqual(0, result);
```

Comment:
Isn't the point here no exception is thrown? I could also get behind returning the numeric value of `vt` as an `int`, which would make this clearer.

Reply:
I thought it should return some non-zero value to indicate that it's doing conversion, rather than using a zero for fallback.
The returned value is in the type of `vt`. Invoke `VariantChangeType` to get the corresponding value?

Reply:
Oh I think I see. This is going to be converting some type and then letting some instance of that type value be converted. Hmmm. Okay, can you share the conversions for each `vt` value? It might be more approprite to be explicit about the return.